### PR TITLE
smooth transitions [WIP]

### PR DESCRIPTION
**Pull request by wzin**

### CODE REVIEW

File: lg_common/extensions/ros_window_ready/README.md [hunk 1]
```diff
@@ -11,4 +11,18 @@ to the `/director/window/ready` ROS topic.
...
+
+Some applications loads assets and do some initialization,
+asynchronously and `onDOMContentLoaded` will generate event
+too early.
+
+To avoid that, add `use_app_event=1` get parameter.
+To send the message broadcast dom event for window:
+
+```javascript
+window.postMessage({ type: "DIRECTOR_WINDOW_READY" }, "*");
```
Comment (wzin) [inline lg_common/extensions/ros_window_ready/README.md]: Oh that's exactly what we've been looking for.

Comment (kiselev-dv) [inline lg_common/extensions/ros_window_ready/README.md]: And here how to send such message from the app
`window.postMessage({ type: "DIRECTOR_WINDOW_READY" }, "*");`

**Comment by kannan4k** [comment]
test

**Comment by wzin** [comment]
test

**Comment by wzin** [comment]
test

**Comment by wzin** [comment]
test

**Comment by wzin** [comment]
test

**Comment by wzin** [comment]
test

**Pull request merged**